src/utils/sgDbProjects.ts

```diff
@@ -23,6 +23,14 @@ export const sgDbCompareKeys: SGSBCompareKey[] = [
   'metadataUri',
```

Comment:
updates to this file are just allowing us to store newly added (and previously missing) Project properties in the DB